src/main/java/com/lorepo/icf/utils/dom/DOMInjector.java

```diff
@@ -28,6 +28,17 @@ public native static void injectJavaScript(String code) /*-{
 	  $wnd.document.getElementsByTagName("head")[0].appendChild(elem);
 	}-*/; 
 
+	public native static void injectLibrary(String url, String libraryName) /*-{
+		if ($wnd.document.getElementById(libraryName)) return;
+		var elem = $wnd.document.createElement("script");
+		elem.setAttribute("src", url);
+		elem.id = libraryName;
+		$wnd.document.getElementsByTagName("head")[0].appendChild(elem);
+		elem.onerror = function () {
+			console.log("External script " + url + "failed to load");
```

Comment:
jeżeli zdecydujesz się na zamienienie z log na error (w icEditor) to tutaj również popraw